package.json

```diff
@@ -2,7 +2,7 @@
   "name": "boltz-frontend",
   "version": "1.0.0",
   "scripts": {
-    "start": "react-scripts start",
+    "start": "BROWSER=none react-scripts start",
```

Comment:
@michael1011 Why not open the app in a browser when launching?

Reply:
In order to test the Joule extension on regtest I am using a browser that is not my default one. My default browser is configured to use Joule with my mainnet node and it annoyed me a little that it opened a new tab in the default browser.

If you want to keep it it's fine too. Just tell me.

Reply:
No, I don't mind just wanted to know.